packages/block-editor/src/components/index.js

```diff
@@ -162,6 +162,7 @@ export { default as __experimentalPublishDateTimePicker } from './publish-date-t
 export { default as __experimentalInspectorPopoverHeader } from './inspector-popover-header';
 export { default as BlockPopover } from './block-popover';
 export { useBlockEditingMode } from './block-editing-mode';
+export { default as TabbedSidebar } from './tabbed-sidebar';
```

Comment:
This is a helpful refactor to work towards unified sidebars, so I don't want to block this progress. That said, I think it will be important to not combine the content of the sidebar with the interaction of the sidebar wrapper. I see the eventual goal as having a shared sidebar component (or another name that does not reference the visual position of the component) with a close button. The content within it can be whatever it needs to be. 

This will be possible if we move to a header title for each sidebar, as proposed in https://github.com/WordPress/gutenberg/pull/61553